### PR TITLE
Issue #613: Create 2020 landing page based on 2019

### PR DESCRIPTION
**Pull request by tai271828**
Init page for PyCon TW 2020 landing page.

### CODE REVIEW

File: src/core/context_processors.py [hunk 1]
```diff
@@ -22,10 +22,10 @@ def pycontw(request):
     lang = get_language()
     if lang and lang.startswith('zh'):
         sponsor_id = '1FAIpQLScEIeCrTHNvwbdNbZt4nK1mteC6NzHtXgF5bVn1KTtR0_sorA'
-        volun_id = '1FAIpQLSeEs59DrqXo3dYS9Cg52sa2MtPurTwpb0XD_s6U7W4HEqYifA'
+        volun_id = '15aD4KP_PywqDrejTIOQyt2SCLTaUX5r21ZhSUkU_fXQ'
```
Comment (yychen) [inline src/core/context_processors.py]: I don't think this Google form ID is valid.

File: src/core/context_processors.py [hunk 2]
```diff
@@ -22,10 +22,10 @@ def pycontw(request):
...
     else:
         sponsor_id = '1FAIpQLScEIeCrTHNvwbdNbZt4nK1mteC6NzHtXgF5bVn1KTtR0_sorA'
-        volun_id = '1FAIpQLScslZSIKPylGvnw2yFNMEINIdoA1avj89FrIfOP2N9H0xbCrg'
+        volun_id = 'D4KP_PywqDrejTIOQyt2SCLTaUX5r21ZhSUkU_fXQ'
```
Comment (yychen) [inline src/core/context_processors.py]: So is this one.

File: src/core/context_processors.py (hunk 1 above)
Comment (tai271828) [reply src/core/context_processors.py]: Thanks for point out. I updated this PR.

File: src/core/context_processors.py (hunk 2 above)
Comment (tai271828) [reply src/core/context_processors.py]: Thanks for point out. I updated this PR.

**Comment by tai271828** [comment]
Thanks for reviewing. I have updated this pull request according to the review comments. Now the pull request is ready for another review.

File: src/pycontw2016/settings/base.py [hunk 3]
```diff
@@ -329,9 +330,8 @@ def node_bin(name):
 )
 
 EVENTS_DAY_NAMES = collections.OrderedDict([
-    (datetime.date(2019, 9, 20), _('Day 1')),
-    (datetime.date(2019, 9, 21), _('Day 2')),
-    (datetime.date(2019, 9, 22), _('Day 3')),
+    (datetime.date(2019, 9, 5), _('Day 1')),
+    (datetime.date(2019, 9, 6), _('Day 2')),
```
Comment (uranusjr) [inline src/pycontw2016/settings/base.py]: Oops

Comment (tai271828) [reply src/pycontw2016/settings/base.py]: Looks like this typo has been fixed in master. : )

**Pull request merged**